RobotFramework_Testsuites/Utils/LibListener.py

```
@@ -80,7 +81,14 @@ def _start_suite(self, data, result):
 
             RobotFramework_Testsuites.CTestsuitesCfg.oConfig.sRootSuiteName = test_suite.name
             RobotFramework_Testsuites.CTestsuitesCfg.oConfig.iTotalTestcases = test_suite.test_count
-            
+
+            if '${localconfig}' in BuiltIn().get_variables()._keys:
+                if re.match('^\s*$', BuiltIn().get_variable_value('${LOCAL_CONFIG}')):
+                    logger.error("local_config input must not be empty!!!")
+                else:
+                    RobotFramework_Testsuites.CTestsuitesCfg.oConfig.lLocalConfig = BuiltIn().get_variable_value('${LOCAL_CONFIG}').strip()
+            elif os.path.isfile(os.environ['ROBOT_LOCAL_CONFIG'] + "/local_config.json"):
```

Hi Son,
here we have a mißunderstanding.
```
elif os.path.isfile(os.environ['ROBOT_LOCAL_CONFIG']:
```
was what I mean. 
1. ```ROBOT_LOCAL_CONFIG``` is optional and by default not existing.
2. if variable ```ROBOT_LOCAL_CONFIG ``` is existing, then we load the content.
3. ```ROBOT_LOCAL_CONFIG``` has to have a fully valid filename.
4. We need to raise an error if ```ROBOT_LOCAL_CONFIG``` has a content, but ```isfile``` fails or the content is not valid.

Please add this update
https://github.com/test-fullautomation/robotframework-testsuitesmanagement/pull/45/files#diff-110f80967316b13e247ab4eb439fdfd960cb2ab8537722ff01bf0cd162e49941
according to our discussion.

Thank you,
 Thomas

---

Hello Thomas,

I will push the commit to modify work follow of local config feature as you mentioned here.

Thank you,
Son